src/components/QandA/QandA.jsx

```diff
@@ -3,60 +3,118 @@ import QuestionList from './QuestionList.jsx';
 import './QAstyles/QandA.css';
 import server from '../../serverRequests.js';
 
-const QandA = ({prod}) => {
+function QandA ({prod}) {
   const [search, setSearch] = useState('');
-  const [count, setCount] = useState(1);
-  const [questions, setQuestions] = useState(['Question_1', 'Question_2', 'Question_3', 'Question_4'])
+  const [qCount, setQCount] = useState(2);
+  const [aCount, setACount] = useState(1);
+  const [productID, setProductID] = useState('');
+  const [answersID, setAnswersID] = useState([]);
+  const [results, setResults] = useState([]);
```

Comment:
What is results representing? If you have questions and answers as state, what does this represent?